### PR TITLE
Minimal Giraffe App

### PR DESCRIPTION
**Pull request by wallymathieu**
From https://github.com/giraffe-fsharp/Giraffe/issues/497#issuecomment-968400589

### CODE REVIEW

**Comment by wallymathieu** [comment]
@TheAngryByrd should this one get squashed or dropped?

File: demo-apps/SampleApp/SampleApp.Tests/SampleApp.Tests.fsproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net7</TargetFramework>
```
Comment (slang25) [inline demo-apps/SampleApp/SampleApp.Tests/SampleApp.Tests.fsproj]:
```suggestion
    <TargetFramework>net7.0</TargetFramework>
```

File: demo-apps/SampleApp/SampleApp/SampleApp.fsproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net7</TargetFramework>
```
Comment (slang25) [inline demo-apps/SampleApp/SampleApp/SampleApp.fsproj]:
```suggestion
    <TargetFramework>net7.0</TargetFramework>
```

File: demo-apps/JwtApp/JwtApp/JwtApp.fsproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net7</TargetFramework>
```
Comment (slang25) [inline demo-apps/JwtApp/JwtApp/JwtApp.fsproj]:
```suggestion
    <TargetFramework>net7.0</TargetFramework>
```

File: demo-apps/IdentityApp/IdentityApp/IdentityApp.fsproj [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net7</TargetFramework>
```
Comment (slang25) [inline demo-apps/IdentityApp/IdentityApp/IdentityApp.fsproj]:
```suggestion
    <TargetFramework>net7.0</TargetFramework>
```

File: demo-apps/GoogleAuthApp/GoogleAuthApp/GoogleAuthApp.fsproj [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net7</TargetFramework>
```
Comment (slang25) [inline demo-apps/GoogleAuthApp/GoogleAuthApp/GoogleAuthApp.fsproj]:
```suggestion
    <TargetFramework>net7.0</TargetFramework>
```

File: appveyor.yml [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
-  - Visual Studio 2017
+  - Visual Studio 2019
   - Ubuntu
 environment:
   DOTNET_CLI_TELEMETRY_OPTOUT: 1
 init:
   - git config --global core.autocrlf true
 install:
-  - ps: .\.psscripts\install-dotnet.ps1 -SdkVersions "3.1.201"
+  - ps: .\.psscripts\install-dotnet.ps1 -SdkVersions "7.0.307"
```
Comment (slang25) [inline appveyor.yml]:
```suggestion
  - ps: .\.psscripts\install-dotnet.ps1 -SdkVersions "7.0.400"
```

Comment (wallymathieu) [reply appveyor.yml]: Nice @slang25 , I think those changes are already merged, so the suggestions should go into a separate PR. I need to rebase this PR in order for it to show only the relevant changes.

Comment (wallymathieu) [reply appveyor.yml]: I've rebased, so the actual changes should be only two files.

**Review by TheAngryByrd — APPROVED** [review]
🎸 Thanks!!

**Pull request merged**